src/org/jitsi/impl/neomedia/rtp/RTPEncodingDesc.java

```diff
@@ -120,6 +125,35 @@ public FrameDesc put(Long key, FrameDesc value)
     };
 
     /**
+     * The {@link TreeMap} that holds the seen {@link FrameDesc}, keyed
+     * by their RTP timestamps.
+     */
+    private final TreeMap<Long, FrameDesc> streamFrames
+        = new TreeMap<Long, FrameDesc>()
+    {
+        /**
+         * A helper {@link LinkedList} that is used to cleanup the map.
+         */
+        private LinkedList<Long> tsl = new LinkedList<>();
+
+        /**
+         * {@inheritDoc}
+         */
```

Comment:
this must be doing something extra than the parent, right? some extra docs here describing what extra logic it has would be good